doc/example-sops.yaml

```diff
@@ -0,0 +1,4 @@
+creation_rules:
+  - path_regex: .*
+    kms: "arn:aws:kms:us-east-1:162808325377:key/5f56d94b-a658-4c92-a226-58bf07c88890"
```

Comment:
Should we have an actual key here?

Reply:
Not sure.  The ARN does not reveal anything about the key.  The key is secret, not the ARN

Reply:
Also, I like to keep the general format/example of something similar in these files so people can more easily understand what should go in there.

Reply:
Right, makes sense.